js/src/lib/interfaces/Libraries.ts

```diff
@@ -226,7 +226,8 @@ from skops.io import load
 
 download("${model.id}", "path_to_folder")
 # make sure model file is in skops format
-# if model is a pickle file, make sure it's from a source you trust
+# if the model is a pickle file, make sure it's from a source you trust
+# and deserialize using the pickle library instead
```

Comment:
Back when I wrote this we decided instead of repetitively changing it we could simply set it for once. I think comment suffices, WDYT?

Reply:
Sure, but something's not working here, as in, when the extension is not `skops`, it shouldn't show this but it is showing it.

Reply:
I feel like it would be cool if we had a section in config.json that states which format the file is serialized with so Hub could use it to infer the snippet. WDYT

Reply:
that's what I suggested in the original PR you had I think, I don't mind that.

Reply:
Any updates on this?

Reply:
@osanseviero there was a renaming around our serialization & loading functions (and a debate around it since planned renaming was causing problems as well) I will make more changes accordingly.

Reply:
Awesome! Thanks a lot! :fire:

Reply:
Can we simply merge this with additional note for now?
We have a [prerequisite issue](https://github.com/skops-dev/skops/issues/228) to determine file format (now that model format is merged) and in model card, we decided for now we depend on file extension to add the code to get started. (so users can depend on that) 
Other option for now is to infer this snippet from file extension which was previously objected. (as far as I remember)